ibis/common/caching.py

```diff
@@ -29,8 +28,11 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+CacheEntry = namedtuple("CacheEntry", ["name", "ref", "finalizer"])
+
+
 class RefCountedCache:
-    """A cache with reference-counted keys.
+    """A cache with (implicitly) reference-counted values.
```

Comment:
```suggestion
    """A cache with implicitly reference-counted values.
```

Seems important enough to not parenthesize!